presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -88,7 +95,7 @@
 {
     public TestHiveLogicalPlanner()
     {
-        super(() -> createQueryRunner(ImmutableList.of(LINE_ITEM), ImmutableMap.of("experimental.pushdown-subfields-enabled", "true"), Optional.empty()));
+        super(() -> createQueryRunner(ImmutableList.of(ORDERS, LINE_ITEM), ImmutableMap.of("experimental.pushdown-subfields-enabled", "true"), Optional.empty()));
```

Comment:
Do we have a planner test in the `presto-hive` module? :open_mouth:

Reply:
I added it to allow testing of filter pushdown and subfield pruning.